react_demo/cpp/index.html

```diff
@@ -0,0 +1,250 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <title>FROM_C_PLUS_PLUS</title>
```

Comment:
Nah, this one was not meant to be public-showcase-clean.

On Saturday, June 6, 2015, John Babak <notifications@github.com> wrote:

> In react_demo/cpp/index.html
> <https://github.com/KnowSheet/SimpleDashboard/pull/7#discussion_r31874075>
> :
>
> > @@ -0,0 +1,250 @@
> > +<!DOCTYPE html>
> > +<html>
> > +  <head>
> > +    <meta charset="utf-8">
> > +    <meta http-equiv="x-ua-compatible" content="ie=edge">
> > +    <title>FROM_C_PLUS_PLUS</title>
>
> If you wanted functional demo, you haven't had to change anything in my
> code, you just had to add file serving and string replacement in the
> index.html file. From having this PR I thought you wanted to make this
> React + C++ code clean enough to be shown to the public.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/KnowSheet/SimpleDashboard/pull/7/files#r31874075>.
>


-- 
And, as always, have a nice day!
Dima

http://dimakorolev.com/ | http://www.linkedin.com/in/dimakorolev |
dmitry.korolev@gmail.com | +1 (312) 593 2783